packages/@react-stately/steplist/src/useStepListState.ts

```diff
@@ -111,11 +118,11 @@ export function useStepListState<T extends object>(props: StepListProps<T>): Ste
   };
 }
 
-function buildKeysMaps<T>(coll: Collection<Node<T>>): { indexMap: Map<Key, number>, keysLinkedList: Map<Key, Key> } {
+function buildKeysMaps<T>(coll: Collection<Node<T>>) {
```

Comment:
You make be able to keep this with a small change, ` { indexMap: Map<Key, number>, keysLinkedList: Map<Key, Key | undefined> }`

Reply:
no need to keep it, it's inferred